### ISSUE TITLE
Missing option for configuration

### ISSUE THREAD

**Issue by Bartlebyy** [crowdin/github-action#2]
We have multiple projects in crowdin that support different languages, and therefore have two configuration files.

So if you could adjust

```
upload_sources() {
  echo "UPLOAD SOURCES";
  crowdin upload sources ${CONFIG_OPTIONS} ${OPTIONS};
}

upload_translations() {
  echo "UPLOAD TRANSLATIONS";
  crowdin upload translations ${CONFIG_OPTIONS} ${OPTIONS};
}
```

to 

```
upload_sources() {
  echo "UPLOAD SOURCES";
  crowdin ${FOO} upload sources ${CONFIG_OPTIONS} ${OPTIONS};
}

upload_translations() {
  echo "UPLOAD TRANSLATIONS";
  crowdin ${FOO} upload translations ${CONFIG_OPTIONS} ${OPTIONS};
}
```

So that I can do FOO="-c crowdin-booking.yml" or FOO="-c crowdin.yml"

**Comment by andrii-bodnar** [comment]
@Bartlebyy Hi, thank you for your suggestion!

Yes, we are going to implement this feature in the nearest future.

**Comment by andrii-bodnar** [comment]
Released in the version [1.0.2](https://github.com/crowdin/github-action/releases/tag/1.0.2).
